cartograph/server/NewMapService.py

```diff
@@ -51,6 +51,69 @@ def filter_tsv(source_dir, target_dir, ids, filename):
                 popularities_writer.writerow(row)
 
 
```

Comment:
Feature request: Could you write a function that checks to see whether a map name is valid (characters, does not exist) and returns an error message or None?